mesonbuild/mtest.py

```diff
@@ -94,6 +94,9 @@ def determine_worker_count() -> int:
     return num_workers
 
 def add_arguments(parser: argparse.ArgumentParser) -> None:
+    parser.add_argument('--maxfail', default=0, type=int,
```

Comment:
Should it be `--max-fail`? Two words separated by a dash, similar to `--print-errorlogs`

Reply:
fwiw, --maxfail is my pytest uses, I think it makes sense to be consistent here. But I've got no strong opinion either way.

Reply:
I originally named it without a dash, for consistency with pytest.

Also, "maxfail" sounds good, "printerrorlogs" doesn't. This is 100% subjective but I stand by my opinion. :D

Reply:
doesn't matter to me either way